includes/configuration_managers/class-wordpress-seo-configuration-manager.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Yoast (WordPress SEO) plugin configuration manager.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+require_once NEWSPACK_ABSPATH . '/includes/configuration_managers/class-configuration-manager.php';
+
+/**
+ * Provide an interface for configuring and querying the configuration of Jetpack.
+ */
+class WordPress_SEO_Configuration_Manager extends Configuration_Manager {
+
+	/**
+	 * The slug of the plugin.
+	 *
+	 * @var string
+	 */
+	public $slug = 'wordpress_seo';
+
+	/**
+	 * Configure Yoast for Newspack use.
+	 *
+	 * @return bool || WP_Error Return true if successful, or WP_Error if not.
+	 */
+	public function configure() {
+		return true;
+	}
+
+	/**
+	 * Is Yoast installed and connected?
+	 *
+	 * @return bool Plugin ready state.
+	 */
+	public function is_configured() {
+		if ( $this->is_active() && class_exists( 'WPSEO_Options' ) ) {
+			return true;
+		}
+		return false;
+	}
+
+	/**
+	 * Get Yoast option.
+	 *
+	 * @param string $key Key of the option to return.
+	 * @param string $default Default value of the option.
+	 * @return string|object Returns value of option.
+	 */
+	public function get_option( $key, $default = '' ) {
+		return \WPSEO_Options::get( $key, $default );
```

Comment:
The get/set option methods should do a check to make sure Yoast is installed and active, otherwise we could have big fatal errors. [I like the Ads configuration manager's approach](https://github.com/Automattic/newspack-plugin/blob/master/includes/configuration_managers/class-newspack-ads-configuration-manager.php#L41-L45).

Reply:
Whoops! https://github.com/Automattic/newspack-plugin/pull/435/commits/66c674360bfcfe7fa5f3d16282874cdb190ebd80